obspy/io/gse2/__init__.py

```diff
@@ -7,6 +7,8 @@
 library `GSE_UTI <ftp://www.orfeus-eu.org/pub/software/conversion/GSE_UTI/>`_
 of Stefan Stange, which is interfaced via Python :mod:`ctypes`.
 
+This module also provides read support for GSE2.0 bulletin format.
+
```

Comment:
Can you maybe also change the very first line in this file to

`obspy.io.gse2 - GSE2/GSE1 and GSE2 bulletin support for ObsPy`

then it will also be reflected in this summary page:

http://docs.obspy.org/pull-requests/1528/packages/index.html